packages/edit-site/src/components/routes/use-title.js

```diff
@@ -47,14 +47,17 @@ export default function useTitle( title ) {
 			document.title = formattedTitle;
 
 			// Announce title on route change for screen readers.
+			if ( ! accessibleTitle ) {
```

Comment:
What's the case where accessibleTitle would be absent? Is it a better option for no announcement if that version of the title isn't made, or fto keep the more verbose announcement?

Reply:
@joedolson It should always be available but I guess I could add a fallback here. The verbose title is so annoying I opted to not use it but easy enough to adjust the conditional.

Reply:
Done.